test/test.js

```diff
@@ -437,6 +437,59 @@ suite('GaiaFastList >', function() {
     });
   });
 
+  suite('images', function() {
+    var el;
+
+    test('it loads images', function() {
+      el = createList();
+      el.configure({
+        getItemImageSrc(item, i) {
+          return `/base/test/lib/artwork-${i % 10}.jpg`;
+        }
+      });
+
+      el.model = createModel();
+
+      var fastList = this.FastList.lastCall.returnValue;
+      return fastList.rendered
+        .then(() => {
+          return Promise.all([].map.call(el.querySelectorAll('img'), img => {
+            return new Promise(resolve => img.onload = resolve);
+          }));
+        });
+    });
+
+    test('it loads correct images when the scrollTop changes', function() {
+      el = createList();
+      el.configure({
+        getItemImageSrc(item, i) {
+          return `/base/test/lib/artwork-${i % 10}.jpg`;
+        }
+      });
+
+      el.model = createModel();
+
+      var fastList = this.FastList.lastCall.returnValue;
+      return fastList.rendered
+        .then(() => {
+          var itemHeight = 60;
+          el.scrollTop = 55 * itemHeight;
+
+          // wait one tick for re-render
+          return Promise.resolve();
+        })
+        .then(() => {
+          var items = el.querySelectorAll('.gfl-item');
+          [].forEach.call(items, item => {
+            var index = item.dataset.index;
+            var expected = `/base/test/lib/artwork-${index % 10}.jpg`;
+            var img = item.querySelector('img');
+            assert.isTrue(!!~img.src.indexOf(expected));
```

Comment:
I just want to check that the abosulte `img.src` contains the original `expected` string. What assertions do you suggest?